Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine:3.21.3
```

Comment:
May as well just use `node:22-slim`? Unless there's a specific reason for using alpine directly.

Reply:
I updated this, good callout